i18n/locales/ca.json

```diff
@@ -1,39 +1,40 @@
 {
-    "components.footer.scrollToTop.button": "Torna al començament",
```

Comment:
I see this file is still updated. Can you revert this file?

Reply:
This file had wrong indentation, should we revert it?

Reply:
Oh, no let's keep the change then. But just don't add the translation key

Reply:
Great, I removed the key